### DISCUSSION TITLE
Is that possible to change the background color to gradient background ?

### DISCUSSION THREAD

**Discussion by medkhalilbk** [tsparticles/react#57] [Q&A] [answered]
Hey contributors! 😎

I'm currently working on a NextJS project, and I'm stuck to customize the React-particles component, I'd like to transform this bit of code:
 
```
background: {
  color: {
    value: "#0d47a1",
  },
},
```
into a sleek linear gradient background like this:
 
`background: linear-gradient(246deg, #D56B97 8.53%, #66228E 84.6%);`

Any tips or ideas are highly appreciated! 🚀

**Comment by matteobruni** [comment, marked as answer]
Background gradients are applied to background-image property https://www.w3schools.com/css/css3_gradients.asp

Set it to `background.image` property instead of `background.color` and it will work 

Sample here
https://github.com/tsparticles/presets/blob/7979797175fad0a31156fce0be83063dc31a33ef/presets/fire/src/options.ts#L38

**Reply by medkhalilbk** [reply]
Thank you for the fast response! 
Much appreciated 🙏
